src/main/java/core/basesyntax/operation/BalanceHandleImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.FruitDataBase;
+import core.basesyntax.model.FruitRecordDto;
+
+public class BalanceHandleImpl implements OperationHandler {
```

Comment:
Зроби те ж саме для інших хендлерів
```suggestion
public class BalanceHandler implements OperationHandler {
```

Reply:
done